app/controllers/assignments_api_controller.rb

```diff
@@ -806,6 +834,10 @@ def get_assignments(user)
 
       preloaded_attachments = api_bulk_load_user_content_attachments(assignments.map(&:description), @context)
 
+      if include_params.include?('statistics')
```

Comment:
Looks like this should be 'score_statistics' instead of 'statistics', right?

Reply:
Yeahhhh that's supposed to be score_statistics. Feel free to make that change - no need to wait for me. I don't know if I somehow messed that up in the most recent push or if it was like that all along, but good catch.